tests/spark_autologging/test_spark_datasource_autologging.py

```diff
@@ -46,6 +46,7 @@ def _get_expected_table_info_row(path, data_format, version=None):
 #   (it is not reset between tests)
 
 
+@pytest.mark.skip(reason='fail on spark 3')
```

Comment:
Can we replace `skip` with `skipif` here?

https://docs.pytest.org/en/stable/reference.html#pytest-mark-skipif